File: exs/mundo_1/java/009.java
/*
Desafio 009
Problema: Faça um programa que leia um número Inteiro qualquer e mostre na
    tela a sua tabuada.
Resolução do problema:
*/
import java.util.Scanner;

class Main{
    public static void main(String[] args){
        Scanner scan = new Scanner(System.in);
        System.out.print("Número: ");
        int num = scan.nextInt();
        System.out.printf("%d x 0 = %d\n", num, num*0);
        System.out.printf("%d x 1 = %d\n", num, num*1);
        System.out.printf("%d x 2 = %d\n", num, num*2);
        System.out.printf("%d x 3 = %d\n", num, num*3);
        System.out.printf("%d x 4 = %d\n", num, num*4);
        System.out.printf("%d x 5 = %d\n", num, num*5);
        System.out.printf("%d x 6 = %d\n", num, num*6);
        System.out.printf("%d x 7 = %d\n", num, num*7);
        System.out.printf("%d x 8 = %d\n", num, num*8);
        System.out.printf("%d x 9 = %d\n", num, num*9);
        System.out.printf("%d x 10 = %d\n", num, num*10);
    }
}

File: exs/mundo_1/golang/009.go
/*
	Desafio: 009
	Problema: Faça um programa que leia um número Inteiro qualquer
    e mostre na tela a sua tabuada.

*/

package main
import "fmt"

func main() {
	var value float64
	fmt.Print("Informe o valor: ")
	fmt.Scan(&value)
	fmt.Println(value, "X 0 =", value * 0)
	fmt.Println(value, "X 1 =", value * 1)
	fmt.Println(value, "X 2 =", value * 2)
	fmt.Println(value, "X 3 =", value * 3)
	fmt.Println(value, "X 4 =", value * 4)
	fmt.Println(value, "X 5 =", value * 5)
	fmt.Println(value, "X 6 =", value * 6)
	fmt.Println(value, "X 7 =", value * 7)
	fmt.Println(value, "X 8 =", value * 8)
	fmt.Println(value, "X 9 =", value * 9)
	fmt.Println(value, "X 10 =", value * 10)
}
